META6.json

```diff
@@ -9,7 +9,7 @@
   "auth": "perl6",
   "depends": [
     "File::Temp",
-    "Pod::To::BigPage:ver<0.5.1+>",
+    "Pod::To::BigPage:ver<0.5.2+>",
```

Comment:
Probably the best would be to remove this dependency. At the end of the day, it's only used to generate a single page, and this is only done infrequently. So please consider just eliminating it, and use documentation to warn about installation.

Reply:
Makes sense. I did so, see 68ea5e97c.